src/main/java/uk/gov/hmcts/reform/idam/health/am/AmHealthProbeConfiguration.java

```diff
@@ -32,7 +32,7 @@ public ScheduledHealthProbeIndicator amIsAliveScheduledHealthProbe(AmIsAliveHeal
     public ScheduledHealthProbeIndicator amPasswordGrantScheduledHealthProbe(AmPasswordGrantHealthProbe amPasswordGrantHealthProbe) {
         return new ScheduledHealthProbeIndicator(
                 amPasswordGrantHealthProbe,
-                HealthProbeFailureHandling.IGNORE,
+                HealthProbeFailureHandling.MARK_AS_DOWN,
```

Comment:
I think this might create a "catch 22": it requires that a service exists but a service cannot be created if the HC returns `DOWN` for AM.

Reply:
@kremi I know exactly where you're coming from but idam-data-provisioning expects AM to be down when it runs. 

The run-data-provisioning-ssl.sh script sets a dns override to bypass the load balancer so that it can run successfully.

https://github.com/hmcts/idam-tools/blob/master/script/run-data-provisioning-ssl.sh#L20-L22

Reply:
Right, thanks @henrydobson !